src/ledger/readme.md

```diff
@@ -72,7 +72,7 @@ graph TD
     D --> E[BlockstoreReader]
 ```
 
-![ShredCollector Component](./imgs/shred_collector_component.png)
+![ShredCollector Component](./imgs/shred_network_component.png)
```

Comment:
I think the actual renaming of the file was missed. So this change probably has lead to a broken link